components/status/ErrorCatchWidget.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <div>
+    <!-- original mockflow ID: id="cmpDd7c4156bb0839a33f5be9502418562bf" -->
+    <div
+      class="div__status-error-catch-background"
+      :style="error_background_cssprops"
+    ></div>
+    <!-- original mockflow ID: id="cmpD94339a6591e3ead19b4308b3a8b0cb68" -->
+    <span class="div_status-error-catch-title-label"
+      >An&nbsp;<wbr />error&nbsp;<wbr />occurred.
+    </span>
+    <!-- original mockflow ID: id="cmpD7504e31fab5b6799a0bbf3fe2514622e" -->
+    <div class="div_status-error-catch-alert-txt">
+      <p>Mantarray software is about to shut down.</p>
+      <p>
+        Please send this log file to
+        <a
+          id="error_contact"
+          href="mailto:contact@curibio.com ? subject = Mantarray Error log"
+          >contact@curibio.com</a
+        >
+      </p>
+    </div>
+    <textarea
+      class="textarea__error-file-path"
+      name="error_file"
+      :rows="compute_number_of_rows"
+      cols="50"
+      spellcheck="false"
+      :value.prop="internal_file_full_path"
+      :style="textarea__error_cssprops"
+    ></textarea>
+    <div class="div__error-button" :style="error_catch_button_cssprops">
+      <ButtonWidget
+        :button_widget_width="450"
+        :button_widget_height="50"
+        :button_widget_top="0"
+        :button_widget_left="0"
+        :button_names="btnnames_create_plate_map_okay"
```

Comment:
for the future, if all these attributes (name, hover colors, enabled status) are never changed by the parent component, you can just put them in as constants the same way you did the width/height. That way we don't clutter up the `created` section, and it's also easier to see how the `ButtonWidget` is configured since the values are all listed directly in the `template` section when looking at the element

Reply:
Implemented